admin/installer/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Helper utilities for CloudFormation Installer's Packer images.
```

Comment:
:grey_question: Given that this module only has 1 submodule in it right now, it sort of makes sense that the docstring for the module is the same as the submodule. That said, it'd be sort of nice if this was slightly more generic so as to suggest what else might be put in this folder.

No action required, but it might make this more maintainable.